README.md

```diff
@@ -29,95 +33,319 @@ are well established for RSE-ops, and which require further development. We can
 OCI reflects an organic, open source and community effort growing to meet the needs of a changing industry landscape. Specifically, Docker [30], by way of being the first prominent,
-commerical container technology and having developed it's own container registry, Docker Hub [31], was an early contributor to these standards. An early version of their RESTful API to interact with container manifests and layers to push or pull containers was adopted into the first distribution spec of OCI, along with an image, runtime, and digest specifications. A container registry or technology is considered OCI compliant if it meets the criteria of these standards. The OCI standards ensured that there was consistency between cloud providers, and that users could translate between them without issue. Representatives from the major industry leaders worked on the standards together, and eventually community members for traditionally non-OCI compliant containers like Singularity joined the effort. While having common standards is considered good practice in the RSE-ops space,
+commerical container technology and having developed its own container registry, Docker Hub [31], was an early contributor to these standards. An early version of their RESTful API to interact with container manifests and layers to push or pull containers was adopted into the first distribution spec of OCI, along with an image, runtime, and digest specifications. A container registry or technology is considered OCI compliant if it meets the criteria of these standards. The OCI standards ensured that there was consistency between cloud providers, and that users could translate between them without issue. Representatives from the major industry leaders worked on the standards together, and eventually community members for traditionally non-OCI compliant containers like Singularity joined the effort. While having common standards is considered good practice in the RSE-ops space,
 because the effort has been primarily industry driven, the needs of more academic or HPC container technologies are not well represented. Academic and high performance computing efforts are arguably siloed, meaning that there are many libraries developed over time that are rooted in a particular lab or institution, and (with a few exceptions)
 groups do not work together. RSE-ops can only be successful if this issue is addressed, meaning that groups from national labs, academic institutions, and other communities
 that rely on HPC figure out how to successfully work together, not just on software but on standards. A good first step would be joining the industry effort to work on OCI standards so that the needs of this different environment are represented in the community, and an equivalent step would be to better collaborate on software projects, training, and best practices.
 
+> yep -- there is a question of how many of these orgs have developers/staff to contribute to
```

Comment:
yeah and sometimes I get the sense that working on them is required (and it's painful lol). Probably that's why change is so hard and slow.